synthtool/gcp/templates/node_library/samples/README.md

```diff
@@ -33,6 +33,16 @@ Before running the samples, make sure you've followed the steps outlined in
 
 `cd ..`
 
+## Run the test
+
+If you want to run all tests of the sample code, you can run the following commands from the root.
+
+`cd samples`
+
```

Comment:
I think you also need to link the local package by running `npm link ../`

We should also write a disclaimer that samples in the generated folder are autogenerated and not tested, thus not guaranteed to work without further tweaking. However, ones in the samples/ are and can be run from the samples directory.